### DISCUSSION TITLE
Language update frequency

### DISCUSSION THREAD

**Discussion by mishari** [streetcomplete/StreetComplete#4016] [Q&A] [answered]
Hi,

I'm wondering approximately how long would it take for localized strings to get released to the users?

Best Regards
Mishari

**Comment by 10992-osm** [comment, marked as answer]
Translations are updated whenever there is a [new release](https://github.com/streetcomplete/StreetComplete/releases).  The [last release](https://github.com/streetcomplete/StreetComplete/releases/tag/v43.0) was 10 days ago, and the [previous stable release](https://github.com/streetcomplete/StreetComplete/releases/tag/v42.0) was at the end of March.  Only [languages with close to 100% translation](https://github.com/streetcomplete/StreetComplete/blob/master/CONTRIBUTING.md#translating-the-app) will be added, and they may be removed if they are not maintained.

**Reply by andrewharvey** [reply]
> Only languages with close to 100% translation will be added, and they may be removed if they are not maintained.

Oh really? For English Australian we only translate a small % of text strings since most are the same as UK English, however I do see the Aussie English localisation coming through to the app.

**Reply by 10992-osm** [reply]
> > Only languages with close to 100% translation will be added, and they may be removed if they are not maintained.
> 
> Oh really? For English Australian we only translate a small % of text strings since most are the same as UK English, however I do see the Aussie English localisation coming through to the app.

As far as I'm aware, it's not an automatic process, so en-AU may have been enabled anyway.  And it would make sense to enable variants of English where translation<100%, since it will still be intelligible if other versions were used.  Although I would have thought that you'd end up seeing American English where en-AU translations are missing, since it appears to be the default.

To be honest, I'm not 100% sure how it works, I'm just going on what I've read in the docs / seen elsewhere.

**Reply by matkoniecz** [reply]
> Oh really? For English Australian we only translate a small % of text strings since most are the same as UK English, however I do see the Aussie English localisation coming through to the app.

That is because it is special as English variant. Not overridden parts are backfilled with default English, resulting in 100% translation.
